doc/specs/stdlib_logger.md

```diff
@@ -8,19 +8,20 @@ title: logger
 ## Introduction
 
 This module defines a derived type, its methods, a variable, and
-constants to be used for the reporting of errors and other
-information. The derived type, `logger_type`, is to be used to define
-both global and local logger variables. The `logger_type` methods serve
-to configure the loggers and use the logger variables to report
-messages to a variable specific list of I/O units termed
-`log_units`. The variable, `global_logger`, of type `logger_type`, is
-intended to serve as the default global logger. The constants serve as
+constants to be used for the reporting of errors, for filtering the 
+log messages, and other information. The derived type, `logger_type`,
+is to be used to define both global and local logger variables. The 
+`logger_type` methods serve to configure the loggers and use the logger
+variables to report messages to a variable specific list of I/O units
+termed `log_units`. The variable, `global_logger`, of type `logger_type`,
+is intended to serve as the default global logger. The constants serve as
 error flags returned by the optional integer `stat` argument.
 
 The logger variables have the option to:
 
 * change which units receive the log messages;
 * report which units receive the log messages;
+* change the level for filtering the log messages;
```

Comment:
Related to my previous comment, an alternative could be `"select which types of messages are logged."`

But I might be mistaken, and the current wording is clear.

Reply:
I also like your sentence. However, I am also not a native English speaker. Any advices from a native English speaker would be nice ;)

Reply:
Ivan-pi's suggestion is good.

Reply:
Changed as proposed.